### PR TITLE
WIP refactor: created reusable color variables

### PR DESCRIPTION
**Pull request by kush-x7**
## About the changes

Made CSS variables of colors in file variables.css and used those variables in all CSS files.

Resolved issue number #308

### CODE REVIEW

File: app/styles/tasks.css [hunk 1]
```diff
@@ -110,7 +100,7 @@ input[type=range]::-moz-range-track {
   border: 0px solid #000000;
 }
 
-input[type=range]::-moz-range-thumb {
+input[type='range']::-moz-range-thumb {
```
Comment (Maheima) [inline app/styles/tasks.css:103]: why are there single quotes on range?

**Review by Maheima — CHANGES_REQUESTED** [review]

File: app/styles/variables.css [hunk 2]
```diff
@@ -1,10 +1,55 @@
...
+  --opaque-black: rgba(0, 0, 0, 0.702);
+
+  --slider-colour: #424cc1;
+  --slider-background: #cfd3ff;
+  --thumb-background: #a1d0ff;
+  --opaque-black: #000000b3;
+
+  --notification-primary-background: #ff6f00;
+  --alert-box-background: #fff8e1;
+  --alert-box-color: #212121;
```
Comment (kush-x7) [inline app/styles/variables.css:54]: **FUTURE WORK:** we should standardize the colors - #311

**Comment by harshith-venkatesh** [comment]
Please ask for review once you change PR from draft mode

File: app/styles/identity.css [hunk 3]
```diff
@@ -1,103 +1,97 @@
...
-    .identity-container {
-        gap: 2rem;
-    }
+  .identity-container {
+    gap: 2rem;
+  }
 
-    .input-div {
-        display: flex;
-        flex-direction: column;
```
Comment (harshith-venkatesh) [inline app/styles/identity.css:83]: Why do we have indentation or padding issue here

Comment (kush-x7) [reply app/styles/identity.css:83]: On saving the code on my local machine it is changing the formatting. Can you please suggest how can I solve this?

Comment (harshith-venkatesh) [reply app/styles/identity.css:83]: It is due to prettier config which has been set globally

**Pull request closed without merging**